sbom-convert.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: sbom-convert
+  version: 0.0.4
+  epoch: 0
+  description: CLI tool based on the protobom library that converts Software Bills of Materials across formats (SPDX and CycloneDX).
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/protobom/sbom-convert
+      tag: v${{package.version}}
+      expected-commit: 0e9c07a02126ec383ed1618aea7386098fdfc542
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+
+  - uses: go/build
+    with:
+      output: sbom-convert
+      packages: .
+
+  - uses: strip
+
```

Comment:
Can you please add a basic test  , here is a simple example of a [test](https://github.com/wolfi-dev/os/blob/main/helm.yaml#L34-L39)

Reply:
Added a simple test that checks the tool version